zh_cn.lang

```diff
@@ -0,0 +1,457 @@
+gui.xaero_preset_fancy=幻想
+gui.xaero_preset_classic=经典
+gui.xaero_preset_social=社交
+gui.xaero_potion_status=药水效果
+gui.xaero_archery_status=显示弓箭剩余数量
+gui.xaero_sprint=更好的冲刺
+gui.xaero_sneak=切换潜行
+gui.xaero_enchants_color=附魔颜色
+gui.xaero_durability=耐久度
+gui.xaero_hunger_setting=饱食度低提醒
+gui.xaero_hp_setting=健康值低提醒
+gui.xaero_explosion_setting=苦力怕预警
+gui.xaero_being_shot_setting=骷髅怪预警
+gui.xaero_xp_setting=经验掉落
+gui.xaero_custom_settings=自定义设置
+gui.xaero_reset_defaults=恢复默认设置
+gui.xaero_quick_use=快速使用物品
+gui.xaero_show_enchants=显示附魔
+gui.xaero_cave_maps=洞穴模式
+gui.xaero_display_teams=显示其他队伍
+gui.xaero_display_waypoints=显示地图上路径点
+gui.xaero_display_players=显示玩家
+gui.xaero_display_mobs=显示怪物
+gui.xaero_display_items=显示物品
+gui.xaero_display_other=显示其他
+gui.xaero_zoom=小地图缩放
+gui.xaero_minimap_size=小地图尺寸
+gui.xaero_entity_amount=实体数量限制
+gui.xaero_black=黑色
+gui.xaero_dark_blue=深蓝色
+gui.xaero_dark_green=墨绿色
+gui.xaero_dark_aqua=深青色
+gui.xaero_dark_red=暗红色
+gui.xaero_dark_purple=深紫色
+gui.xaero_gold=金色
+gui.xaero_gray=灰色
+gui.xaero_dark_gray=深灰色
+gui.xaero_blue=蓝色
+gui.xaero_green=绿色
+gui.xaero_aqua=青色
+gui.xaero_red=红色
+gui.xaero_purple=亮紫色
+gui.xaero_yellow=黄色
+gui.xaero_white=白色
+gui.xaero_percent=百分比
+gui.xaero_numeral=数值
+gui.xaero_both=都
+gui.xaero_tiny=极小
+gui.xaero_small=小
+gui.xaero_medium=中
+gui.xaero_large=大
+gui.xaero_pvp_sprint=PvP 冲刺
+gui.xaero_quick_eat=快速进食
+gui.xaero_toggle_sneak=切换潜行
+gui.xaero_quick_use_1=快速使用 1
+gui.xaero_quick_use_2=快速使用 2
+gui.xaero_quick_use_3=快速使用 3
+gui.xaero_quick_use_4=快速使用 4
+gui.xaero_quick_use_5=快速使用 5
+gui.xaero_quick_use_6=快速使用 6
+gui.xaero_quick_use_7=快速使用 7
+gui.xaero_quick_use_8=快速使用 8
+gui.xaero_quick_use_9=快速使用 9
+gui.xaero_zoom_in=小地图拉近
+gui.xaero_zoom_out=小地图拉远
+gui.xaero_on=开启
+gui.xaero_off=关闭
+gui.xaero_restores=还原
+gui.xaero_preset_topleft=左上角
+gui.xaero_preset_topright=右上角
+gui.xaero_waypoint_teleport=传送
+gui.xaero_waypoints_key=打开路径点屏幕
+gui.xaero_display_coords=显示坐标
+gui.xaero_disable_enable=禁用/启用
+gui.xaero_disabled=禁用
+gui.xaero_minimap_guide=左键按住并拖动来移动小地图。
+gui.xaero_change_position=更改位置
+gui.xaero_preset_bottom_left=左下角
+gui.xaero_preset_bottom_right=右下角
+gui.xaero_ingame_waypoints=游戏内路径点
+gui.xaero_next=下一页 >>
+gui.xaero_previous=<< 上一页
+gui.xaero_lock_north=小地图锁定北方
+gui.xaero_deathpoint=上次死亡地点
+gui.xaero_deathpoint_old=此前死亡地点
+gui.xaero_deathpoints=死亡地点
+gui.xaero_chunkgrid=区块网格颜色
+gui.xaero_entity_colours=实体颜色
+gui.xaero_players=玩家
+gui.xaero_mobs=友善生物
+gui.xaero_hostile=敌对生物
+gui.xaero_items=物品
+gui.xaero_other=其他实体
+gui.xaero_hunger_low=饱食度低
+gui.xaero_hp_low=健康值低
+gui.xaero_enlarge_map=展开小地图
+gui.xaero_fbo=Beta
+gui.xaero_potion_status_settings=药水状态设置
+gui.xaero_potion_names=药水效果名称
+gui.xaero_opacity=不透明度
+gui.xaero_slime_chunks=显示史莱姆区块
+gui.xaero_unlimited=无限制
+gui.xaero_old_deathpoints=保留旧的死亡点
+gui.xaero_toggle_map=小地图开关
+gui.xaero_antialiasing=抗锯齿
+gui.xaero_safe_mode=安全模式
+gui.xaero_transfer=转移
+gui.xaero_transfer_all=转移所有路径点
+gui.xaero_clear=清除集合
+gui.xaero_from=从
+gui.xaero_to=到
```

Comment:
Not sure about above 2 lines.

Reply:
They seem correct to me when translating back, but you know better. Have you decided anything on these?

Reply:
Oh, if you need context, I'm pretty sure these are used on the waypoint transfer screen in the waypoints menu -> options -> transfer.

Reply:
This is correctly translated